.github/workflows/gradle-wrapper-validation.yml

```diff
@@ -1,14 +1,14 @@
 name: Gradle wrapper validation
+
 on:
-  pull_request:
-    paths:
-      - '**/gradle/wrapper/**'
   push:
-    paths:
-      - '**/gradle/wrapper/**'
+  pull_request:
+
+permissions:
+  contents: read
```

Comment:
github action permissions may be leaking into unrelated PRs as I copy things over from the instrumentation repo, but it will just mean less work for me later when it comes to fixing OSSF Scorecard Token-Permissions score which is currently 0/10